python/ray/actor.py

```diff
@@ -545,7 +546,7 @@ def _remote(self,
             meta.language,
             meta.actor_creation_function_descriptor,
             creation_args,
-            meta.max_reconstructions,
```

Comment:
This means we have max_reconstructions turned on by default before? I don't think that happened. What's `meta.max_reconstructions`?

Reply:
`meta.max_reconstructions` is just whatever was set in the decorator. This change is just allowing `max_reconstructions` to be passed using `options()`